app/javascript/components/tooltip/component.spec.js

```diff
@@ -0,0 +1,30 @@
+import { Tooltip as MuiTooltip } from "@material-ui/core";
+
+import { mountedComponent, screen } from "../../test-utils";
+
+import Tooltip from "./component";
+
+describe("components/tooltip", () => {
+  it("renders tooltip", () => {
+    const TooltipComponent = () => (
+      <Tooltip title="wrapper content" data-testid="wrapper-content">
+        <div>wrapped children</div>
+      </Tooltip>
+    );
+    mountedComponent(<TooltipComponent />);
+
+    expect(screen.queryAllByRole('tooltip')).toHaveLength(0);
```

Comment:
this is not testing if the title is present. currently this is equal to the next test